security/stunnel/src/opnsense/mvc/app/models/OPNsense/Stunnel/Stunnel.xml

```diff
@@ -45,6 +45,16 @@
                     <ValidationMessage>port needs to be an integer value between 1 and 65535</ValidationMessage>
                     <Required>Y</Required>
                 </connect_port>
+                <protocol type="OptionField">
+                    <default>none</default>
+                    <OptionValues>
+                        <none>not set [default]</none>
+                        <imap>IMAP</imap>
+                        <pop3>POP3</pop3>
+                        <smtp>SMTP</smtp>
+                    </OptionValues>
+                    <Required>Y</Required>
```

Comment:
it's probably easier to set required to N and ditch the `none` value, since this will need a content migration (although automatic after setting a new version, which is needed anyway, the none option itself doesn't really add anything here)

Reply:
OK.